### PR TITLE
Move base models from zouwu to automl

### PR DESCRIPTION
**Pull request by yushan111**
This PR solves comment in PR #3647 
>We should keep `BaseModel`, `PytorchBaseModel` and `KerasBaseModel` in `automl\model` folder

The PR also moves `model_builder` back to `automl\model` folder

### CODE REVIEW

File: pyzoo/zoo/zouwu/model/forecast/model/XGBoost.py [hunk 1]
```diff
@@ -19,7 +19,7 @@
 
 from xgboost.sklearn import XGBClassifier
 from zoo.automl.common.metrics import Evaluator
-from zoo.zouwu.model.forecast.model.abstract import BaseModel
+from zoo.automl.model.abstract import BaseModel
```
Comment (jason-dai) [inline pyzoo/zoo/zouwu/model/forecast/model/XGBoost.py:23]: this file will go to `orca/automl/xgboost`?

**Comment by jason-dai** [comment]
`zouwu/model/forecast/model` is probably too deep. I think we can put all the basic models under `zouwu/model/`, and put all the forecasters under `zouwu/model/forecast`

**Comment by yushan111** [comment]
> `zouwu/model/forecast/model` is probably too deep. I think we can put all the basic models under `zouwu/model/`, and put all the forecasters under `zouwu/model/forecast`

Got it. I will open a new PR.

**Comment by yushan111** [comment]
Jenkins: http://10.239.47.210:18888/view/ZOO-PR/job/ZOO-PR-Validation/5302/

**Pull request merged**